src/routing/error_cause.rs

```diff
@@ -64,7 +64,7 @@ pub enum ErrorCause {
     CanisterIdNotResolved,
     CanisterRouteNotFound,
     SubnetNotFound,
-    SubnetUnavailable,
+    SubnetUnavailable(String),
```

Comment:
Maybe we can put principal there?

Reply:
Yes, I started with principal and then realized that we get it as a string from the BN response metadata. That's why I switched. This thing is anyway a temporary error page for the vetkey production key back next week.